internal/common/db.go

```diff
@@ -537,3 +538,10 @@ func CloseDB(db *gorm.DB) {
 	}
 	_ = sqlDB.Close()
 }
+
+func EscapePlaceHolders(message string) string {
```

Comment:
Can this be removed? I don't see it being used anywhere and it doesn't have tests.

Reply:
Yes, I will remove it